samples/azure-quantum/entanglement-swap/ES-quantinuum-qsharp.ipynb

```diff
@@ -0,0 +1,1120 @@
+    "If Alice now wishes to transfer quantum information, she will entangle some payload with her qubit and measure her qubit and payload.She will transmit the results of the measurement to Bob, who can use them to reconstruct Alice's payload on his entangled qubit.\n",
+    "\n",
+    "In the following we code will implement this operation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/x-qsharp-data": "[]",
+      "text/html": [
+       "<ul></ul>"
+      ],
+      "text/plain": []
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "open Microsoft.Quantum.Diagnostics;"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/x-qsharp-data": "[\"CreateBellState\",\"Send\",\"Receive\"]",
+      "text/html": [
+       "<ul><li>CreateBellState</li><li>Send</li><li>Receive</li></ul>"
+      ],
+      "text/plain": [
+       "CreateBellState, Send, Receive"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "operation CreateBellState(alice : Qubit, bob : Qubit) : Unit is Adj + Ctl {\n",
+    "    H(alice);\n",
+    "    CNOT(alice, bob);\n",
+    "}\n",
+    "\n",
+    "operation Send(alice : Qubit, payload: Qubit) : (Result, Result) {\n",
+    "    CNOT(payload, alice);\n",
+    "    H(payload);\n",
+    "    return (M(alice), M(payload));\n",
+    "}\n",
+    "\n",
+    "operation Receive(bob : Qubit, (cr_x : Result, cr_z : Result)) : Unit {\n",
+    "    if (cr_x == One) { X(bob); }\n",
+    "    if (cr_z == One) { Z(bob); }\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will now use Q# visualization tools to show the unitary responsible from creating the Bell state.\n",
+    "Printing unitary transformations implemented by operations is a great way to check whether your implementation is mathematically correct (for small operations)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/x-qsharp-data": "[\"ShowBellstate\",\"_b35e7070b5ea46b2bc578f8fd14a6602_ShowBellstate\"]",
+      "text/html": [
+       "<ul><li>ShowBellstate</li><li>_b35e7070b5ea46b2bc578f8fd14a6602_ShowBellstate</li></ul>"
+      ],
+      "text/plain": [
+       "ShowBellstate, _b35e7070b5ea46b2bc578f8fd14a6602_ShowBellstate"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "operation ShowBellstate() : Unit {\n",
```

Comment:
Nit, but you may want `Dump` instead of `Show` here; similarly, `state` should be `State` to match `CreateBellState`.